test/enums.jl

```diff
@@ -81,59 +81,20 @@ end
 @test_throws ArgumentError eval(:(@enum Test22 1=2))
 
 # other Integer types of enum members
-# TODO - not yet supported
```

Comment:
A lot of these `.val` related tests don't really make sense with bitstype-based enums.